glean-core/ffi/src/uuid.rs

```diff
@@ -13,6 +13,7 @@ use crate::{
 
 define_metric!(UuidMetric => UUID_METRICS {
     new           -> glean_new_uuid_metric(),
+    test_get_num_recorded_errors -> glean_uuid_test_get_num_recorded_errors,
```

Comment:
Can you remove the `dead_code` todo in `rlb/src/private/uuid.rs` now that `test_get_num_recorded_errors` is used?